psydac/core/bsplines.py

```diff
@@ -78,7 +82,7 @@ def find_span(knots, degree, x):
         Knot span index.
     """
     x = float(x)
-    knots = np.asarray(knots, dtype=float)
+    knots = np.asarray(knots, dtype=np.float64)
```

Comment:
The idea was to avoid potential cross platforms problems like the one with int being treated differently on windows. It also is clearer that the floats are indeed using 64 bits.

Reply:
Changed in [0aaf755](https://github.com/pyccel/psydac/pull/214/commits/0aaf755ef4c7f10dfbd384e90756f4ecec080fd0)